lib/OpenQA/Setup.pm

```diff
@@ -163,14 +167,17 @@ sub read_config {
 
     for my $section (sort keys %defaults) {
         my @known_keys = sort keys %{$defaults{$section}};
+
         # if no known_keys defined - just assign every key from the section
         if (!@known_keys && $cfg) {
             for my $k ($cfg->Parameters($section)) {
                 $app->config->{$section}->{$k} = $cfg->val($section, $k);
             }
         }
+
         for my $k (@known_keys) {
             my $v = $cfg && $cfg->val($section, $k);
+
```

Comment:
please don't introduce unrelated whitespace changes. These are easy to introduce useless conflicts, make reviewing harder and are noisy. We are happy to accept whitespace changes but then please make them explicit in their own commits with a proper reasoning and at best applying the same rules to all relevant files at the same time.

Reply:
sure thing. i will fix